### PR TITLE
Updated Maps

### LINKED ISSUE #270: Pilots with no vehicle perms sahrani
**Issue**
Buffalo pilots cannot enter UH-60

**Solution**
change class name from whatever it is now to the correct class name (5 minute fix)

### LINKED ISSUE #271: it's possible to activate boxes extremely quickly by flying there and immediately landing onto the objective, then activating the box

### LINKED ISSUE #272: wrong vehicle classes
Wrong vehicle classnames means that permissions isn't being applied to some vehicles, namely Saber and the Blackhawk

### LINKED ISSUE #273: simulation disabled on medic spawn
sahrani

### LINKED ISSUE #274: Hunter perms "no workie" - random pubber
sahrani

### LINKED ISSUE #275: add the new strikers
all maps

### LINKED ISSUE #276: Strong points spawn too many enemies
all maps

### LINKED ISSUE #277: Add working air commander
all maps

### LINKED ISSUE #278: Altis fails to load
altis

### LINKED ISSUE #279: CSAT spawning
takistan

### LINKED ISSUE #280: Apollo area rework
takistan

### PR DESCRIPTION
**Pull request by davidcit646**
**General**
resolves #277
resolves #275
resolves #271
resolves #276

**Altis**
resolves #278

**Sahrani**
resolves #270
resolves #272
resolves #273
resolves #274

**Takistan**
resolves #279
resolves #280

### CODE REVIEW

**Review by 7cavZlobinN — APPROVED** [review]

**Comment by BojanOnTheInternet** [comment]
Do not merge until my approval. I'll be making changes to remove the WFW missions from the auto build script so that we can manually manage those missions. Expect that change either today or tomorrow.

**Comment by BojanOnTheInternet** [comment]
Should be good to merge this once all features are added - I've removed WFW from the build scripts

**Pull request merged**
